scio-schemas/src/main/protobuf/simple_v2.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto2";
+
+option java_package = "com.spotify.scio.proto";
+option optimize_for = SPEED;
+
+message SimplePB {
```

Comment:
Just curious, why `SimpleV2.SimplePB` and `Track.TrackPB`, not just something like `TrackV1` & `TrackV2`?